### DISCUSSION TITLE
[v4] .svelte files not being picked up tailwind/vite

### DISCUSSION THREAD

**Discussion by sgcullen** [tailwindlabs/tailwindcss#15393] [General]
Hi Team,
I'm struggling to get v4 working with a vite+svelte app.
The steps I've followed the various steps:

1. Importing and including `plugins: [tailwindcss()]` in the vite plugins array
2. Adding `@import tailwindcss` to the app.css

Perhaps this information isn't enough but any ideas hugely appreciated :-)

**Comment by jessezwd** [comment]
Same problem.  Maybe missing some steps?

**Comment by adamwathan** [comment]
Hey! Can you create a repo with a really simple app where it isn’t working so we can look?

**Reply by jessezwd** [reply]
An empty svelte project with twv4 configuration: https://github.com/jessezwd/dwv4

**Reply by sgcullen** [reply]
Thanks for creating this @jessezwd 
@adamwathan - let me know if I can further assist diagnose

**Reply by philipp-spiess** [reply]
@jessezwd I tried our your project and it's missing importing the CSS file from the root of your app. That's step 6 on from the docs page: https://tailwindcss.com/docs/guides/sveltekit

Once I added the following file, the project works as expected:

<img width="356" alt="Screenshot 2024-12-17 at 10 21 07" src="https://github.com/user-attachments/assets/135dc62d-a993-4af1-b314-47ae75ba19dc" />

<img width="696" alt="Screenshot 2024-12-17 at 10 21 32" src="https://github.com/user-attachments/assets/64de998a-ee59-4ba6-9fae-7e9fc8d8c9ee" />

**Reply by jessezwd** [reply]
Thanks a lot!
